fixtures-service/generate_graphql_responses.py

```diff
@@ -0,0 +1,35 @@
+from responses_generator import ShopCategoriesGenerator, ShopListsGenerator, ProductListsGenerator
+
+import argparse
+
+
+def generate_responses_for_variant(output_folder, variant):
+
+    generator = ShopListsGenerator(output_folder, variant)
+    generator.generate()
+
+    generator = ShopCategoriesGenerator(output_folder, variant)
+    generator.generate()
+
+    generator = ProductListsGenerator(output_folder, variant)
+    generator.generate()
+
+
+def main(output_folder, fixture_variant):
+    if fixture_variant == 'all':
+        for variant in 'tiny', 'small', 'medium', 'large':
```

Comment:
The identifier `variant` stands for two different things : a fixture variant (big, medium, ....) and a product variant in the context of our business logic ... those kind of ambiguities should be minimized. Pleise, choose something else instead of fixture "variant".

Reply:
We could agree on "fixture set".